engine/server/buildkitcontroller.go

```diff
@@ -257,7 +257,7 @@ func (e *BuildkitController) Session(stream controlapi.Control_SessionServer) (r
 
 		labels := opts.Labels
 		labels = append(labels, pipeline.EngineLabel(e.EngineName))
-		labels = append(labels, pipeline.LoadServerLabels(engine.Version, runtime.GOOS, runtime.GOARCH)...)
+		labels = append(labels, pipeline.LoadServerLabels(engine.Version, runtime.GOOS, runtime.GOARCH, e.cacheManager.ID() != "local")...)
```

Comment:
Since we're relying on it now, we should make "local" a const in `cache/manager.go` and use it there and here. Otherwise I can foresee this very easily being overlooked if we change things in the future.